src/generators/npm-install.ts

```diff
@@ -0,0 +1,30 @@
+import { readPackageJSON } from "pkg-types";
+import { codeBlock } from "omark";
+import { defineGenerator } from "../generator";
+
+export default defineGenerator({
+  name: "npm-install",
+  async generate({ options, args }) {
+    const pkg = await readPackageJSON(options.dir);
+    const name = args.name || pkg.name || "package-name";
+    const dev = !!args.dev;
+    const pkgInstalls = [
+      ["npm", "install", "--save-dev"],
```

Comment:
I think `-D` is supported as well in modern npm

Reply:
Should we add `nypm`? Just stick to the basics?

Reply:
Hmm that's a nice idea to also list nypm!

Reply:
I could make it how we did `unjs/docs` as a param to auto (default to true) if you want to disable you can.